variables.tf

```diff
@@ -13,6 +13,16 @@ variable "azure_location" {
   type        = string
 }
 
+variable "key_vault_access_ipv4" {
+  description = "List of IPv4 Addresses that are permitted to access the Key Vault"
+  type        = list(string)
+}
+
+variable "tfvars_filename" {
+  description = "tfvars filename. This file is uploaded and stored encrupted within Key Vault, to ensure that the latest tfvars are stored in a shared place."
```

Comment:
`encrypted` not encrupted

Reply:
Welp ... Think we have other places to fix that too Lol ...
https://github.com/search?q=org%3ADFE-Digital%20encrupted&type=code